### ISSUE TITLE
Let's add a little .env support

### ISSUE THREAD

**Issue by mattcreager** [hunterloftis/docker-plugin#8]
Foreman automatically parsed config vars from the .env file and made them available to the processes being kicked-off.

Should the docker-plugin provide similar functionality?

**Comment by hunterloftis** [comment]
Good idea Matt, it should probably handle .env files in a similar way. That may be a little further down the implementation; leaving open.
